c/blake3_dispatch.c

```diff
@@ -223,6 +223,23 @@ void blake3_compress_xof(const uint32_t cv[8],
   blake3_compress_xof_portable(cv, block, block_len, counter, flags, out);
 }
 
+
+void blake3_xof_many(const uint32_t cv[8],
+                     const uint8_t block[BLAKE3_BLOCK_LEN],
+                     uint8_t block_len, uint64_t counter, uint8_t flags,
+                     uint8_t out[64], size_t outblocks) {
+#if defined(IS_X86)
+  const enum cpu_feature features = get_cpu_features();
+#if defined(__unix__) && !defined(BLAKE3_NO_AVX512)
```

Comment:
@sneves what do you think about using `__unix__` here?

Reply:
Considering that what is effectively being excluded here is Windows, maybe `!defined(_WIN32)` would make more sense. I believe Apple toolchains may not respect `__unix__` and will define `__APPLE__` or  `__MACH__` or something like that instead.

Reply:
Done. 4cc34b0e85002f6fef3cd8b264343172cfbc9f58